docs/api/app.md

```diff
@@ -348,6 +348,25 @@ bar, and on OS X you can visit it from dock menu.
 
 Clears the recent documents list.
 
+### `app.setAsDefaultProtocolClient(protocol)` _OS X_ _Windows_
```

Comment:
On both systems, a different app can always overwrite. On OS X, simply removing would involve a bit of hackery (we could call the set with an invalid bundle is); on Windows, we could also delete or overwrite the key. Basically the same it works with magnet links, phone links, or other non-restricted uri schemes.

Both systems don't really have a way to simply "unset" custom uri schemes (in a way that would restore previous handlers, for instance), so I'm inclined to not offer an API that makes Electron users *think* that we can really offer that reliably.

I trust your opinion though, I could easily implement an "overwrite with nonsense" API.